src/language-css/print/misc.js

```diff
@@ -27,6 +27,16 @@ function adjustStrings(value, options) {
 
 function quoteAttributeValue(value, options) {
   const quote = options.singleQuote ? "'" : '"';
+
+  // Check if the value ends with `s`, `i` can be solved by postcss
+  if (/\s+s$/u.test(value)) {
+    value = value.replace(/^(.+?)\s+(s)$/u, (_, content, flag) =>
```

Comment:
Can you use named group here?

`/^(?<content>.+?)\s+(?<flag>s)$/u`

Maybe this branch can return directly?